django/core/management/base.py

```diff
@@ -244,18 +244,20 @@ def get_version(self):
         """
         return django.get_version()
 
-    def create_parser(self, prog_name, subcommand):
+    def create_parser(self, prog_name, subcommand, **kwargs):
         """
         Create and return the ``ArgumentParser`` which will be used to
-        parse the arguments to this command.
+        parse the arguments to this command. Hi.
```

Comment:
Remove this.

Reply:
Yes, of course. Sorry about that. I had some issues, and did some tests.
I'll fix this mess.